service/service.go

```diff
@@ -217,13 +251,17 @@ func (svc *Service) Close(ctx context.Context) error {
 }
 
 // registerCheckers adds the checkers for the service clients to the health check object.
-func (svc *Service) registerCheckers() error {
+func (svc *Service) registerCheckers() (err error) {
 	hc := svc.healthCheck
 
 	if err := hc.AddCheck("Kafka consumer", svc.consumer.Checker); err != nil {
 		return fmt.Errorf("error adding check for Kafka: %w", err)
```

Comment:
Also just noticed might be good to add 'consumer' to the end of this error message now to easily tell